connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfigTransformer.java

```diff
@@ -46,7 +52,15 @@ public WorkerConfigTransformer(Worker worker, Map<String, ConfigProvider> config
         if (configs == null) return null;
         ConfigTransformerResult result = configTransformer.transform(configs);
         if (connectorName != null) {
-            scheduleReload(connectorName, result.ttls());
+            String action = configs.get(ConnectorConfig.CONFIG_RELOAD_ACTION_CONFIG);
```

Comment:
You might consider using any `public static` utilities in `ConfigDef`/`AbstractConfig` to help with this here. I know at least `ConfigDef.parseType` could help here since it also does things like trim the string. Reusing that code will keep this closer to the normal behavior of `ConfigDef`s (and if we eventually move this functionality to be part of `ConfigDef` itself, will probably make it a simpler transition).

Reply:
Thanks for the suggestion, I'm using `ConfigDef.parseType` now.